### DISCUSSION TITLE
Battlezone 98 Redux - Community Bug List

### DISCUSSION THREAD

**Discussion by LordBramster** [BattlezoneScrapField/.github#1] [Battlezone Games Bug Lists]
> Original Poster: grizzlyone95

# Source/Engine Fixes
- Map list refreshing and jumping to top
- Netcode lag is awful, regularly ending games half way through. Herp's mod does help for this, at the very least it should be included in base game.
- Campaign end of mission screen broken
- Chunks not visible. Add an ODF flag to allow custom effects, rather than modifying Ogre to show these as a quick solution?
- Jump sniping to 1.5 behavior
- perceivedTeam bug
- Armory glitches in multiplayer (currently fixed via Lua mod)
- expose GetWeaponMask to lua
- You cannot deploy soldiers when targeting a unit with a PerceivedTeam that is considered friendly to your team.

# Asset Fixes
These can be done by the community (through mods) but would prefered to be in the base game:
- Walker cockpits jumping
- Turret cockpits simply not functioning, unable to aim in first person
- Add upgraded HD reticles from blue banana

**Comment by GrizzlyOne95** [comment]
This is excellent, thank you. Can you add me as an editor please? I have an updated list.

**Reply by LordBramster** [reply]
Sure! I can do it when I'm next available 😄
